language/move-binary-format/src/control_flow_graph.rs

```diff
@@ -46,12 +50,21 @@ struct BasicBlock {
 pub struct VMControlFlowGraph {
     /// The basic blocks
     blocks: Map<BlockId, BasicBlock>,
+    /// Basic block ordering for traversal
+    traversal_successors: Map<BlockId, BlockId>,
+    /// Map from blocks with a loop's last continue to that loop's starts
+    loop_last_continue_blocks: Map<BlockId, BlockId>,
+}
+
+pub struct LoopBounds {
+    pub loop_start: CodeOffset,
```

Comment:
Is a loop start the same as a [loop head](https://en.wikipedia.org/wiki/Control-flow_graph#:~:text=A%20loop%20header%20(sometimes%20called,has%20no%20%22loop%20header%22.)? And if not, how are you defining it (again, in terms of nodes and edges in the CFG)?

Reply:
It is a `block_start` that is also a loop head. I went with this since the term `start` was used throughout the module